test/run

```diff
@@ -111,9 +117,14 @@ if [ -n "${TEST_COMPOSE-}" ] && [ "$TESTING_BRANCH" == "main" ]; then
     fi
 
     mkdir -p ~/.openidc
-    cp $COCKPIT_FEDORA_WIKI_STAGING_TOKEN ~/.openidc/oidc_wikitcms.json
+    if [ -n "${TEST_FEDORA_WIKI_STAGING-}" ]; then
```

Comment:
Please double-check this. This check means "variable is defined", but that's not what you do in line 39/42. Are you sure you don't mean `if [ "${TEST_FEDORA_WIKI_STAGING-}" = "true" ]`?

Reply:
Did the change to:
 if [ "$TEST_FEDORA_WIKI_STAGING" == "true" ]; then

It's always defined for COMPOSE scenario.

Reply:
Your suggestion was put in place in the end :)